services/wallet/router/router_v2.go

```diff
@@ -294,98 +322,141 @@ func findBestV2(routes [][]*PathV2, tokenPrice float64, nativeChainTokenPrice fl
 func validateInputData(input *RouteInputParams) error {
```

Comment:
@IvanBelyakoff tests for `validateInputData` will be covered in this PR which should be merged soon https://github.com/status-im/status-go/pull/5323 also if we see that anything is missed/overlooked there, we will add, but the purpose of it is to not let invalid input params to get to the resolveCandidates or any other function below it.

> Option 2 would be...

This sounds ok to me, we can implement that within this issue https://github.com/status-im/status-desktop/issues/14849 and remove all those `if params.TestsMode` checks.

Reply:
> 5^20 is like 95 quadrillion combinations. This does not look fine to me

Well, if we take into consideration all possible tokens that exist now, it's way more... hahahahha

So, as said above `validateInputData` will have its own tests, so any data coming to functions below should be considered valid since those are not functions exposed via API.
What we really need to check is a combination of 3 different chains for `DisabledFromChainIDs` and `DisabledToChainIDs` for ETH and any erc20 token, let's say `USDC`, that's all, which is a waaaay less than those quadrillions you mentioned.